src/example-types/metricGroups/dateRangesGroupStats.js

```diff
@@ -11,7 +11,10 @@ let buildGroupQuery = ({ field, ranges }, children, groupingType) => ({
 })
 
 let drilldown = ({ field, drilldown }) => {
-  let [gte, lt] = _.split('-', drilldown)
+  let separatorIndex = _.floor(drilldown.length / 2)
+  let gte = drilldown.slice(0, separatorIndex)
```

Comment:
The reason I'm slicing like this is because the drilldown is being provided as two dates joined by a hyphen, but they are defaulted to strict date time which results in a string like: `2022-08-02T05:00:00.000Z-2022-10-02T05:00:00.000Z` which doesn't have a straightforward way to split by a dividing character.  So given they are two dates separated by "something" it seemed the most consistent to just get the index of the separator and slice based on that.